README.md

```diff
@@ -15,24 +15,25 @@ mvn clean install
-| AWS_URL                                         | https://s3.amazonaws.com | end point for the S3 bucket                                    |
-| USE_AWS_SDK                                     | false                    | Use the AWS SDK for S3 access                                  |                     |
-| S3_ACCESS_KEY_ID                                | -                        | access key for the S3 bucket                                   |
-| S3_SECRET_ACCESS_KEY                            | -                        | secret key for the S3 bucket                                   |
-| spring.task.execution.pool.max-size             | 10                       | refresh/export thread pool's max size                          |
-| mod-lists.general.refresh-query-timeout-minutes | 90                       | Max time to wait for an FQL query to run during a list refresh |
+| Name                                             | Default Value            | Description                                                    |
+|--------------------------------------------------|--------------------------|----------------------------------------------------------------|
+| DB_HOST                                          | localhost                | Postgres hostname                                              |
+| DB_PORT                                          | 5432                     | Postgres port                                                  |
+| DB_USERNAME                                      | postgres                 | Postgres username                                              |
+| DB_PASSWORD                                      | postgres                 | Postgres password                                              |
+| DB_DATABASE                                      | postgres                 | Postgres database name                                         |
+| server.port                                      | 8081                     | Server port                                                    |
+| MAX_LIST_SIZE                                    | 1250000                  | max size of each list                                          |
+| LIST_EXPORT_BATCH_SIZE                           | 1000                     | batch size for exports                                         |
+| LIST_APP_S3_BUCKET                               | list-exports-us-west-2   | Name of the S3 bucket for exports                              |
+| AWS_REGION                                       | us-west-2                | region of the S3 bucket                                        |
+| AWS_URL                                          | https://s3.amazonaws.com | end point for the S3 bucket                                    |
+| USE_AWS_SDK                                      | false                    | Use the AWS SDK for S3 access                                  |                     |
```

Comment:
if we're making this big diff can you remove the extra `|` hanging out over there?

Reply:
Good call. I didn't even see that